x/programs/cmd/simulator/cmd/config_test.go

```diff
@@ -29,7 +30,9 @@ func TestValidateAssertion(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		result, err := validateAssertion(tt.actual, &tt.assertion)
+		actual, err := borsh.Serialize(tt.actual)
+		require.NoError(err)
```

Comment:
It's much nicer being more explicit about the encoding, thanks!

Reply:
`go test` was not compiling before using `borsh.Serialize` I think simulator go tests are not run by the CI.
Not sure to understander your comment. Are you just saying it is better right now, or are you asking to be more explicit? 😀

Reply:
It was a CI issue, some static checks were not done and it led me to think that go was inferring the type, it was an actual bug that was apparently ignored! I'm not asking you to change anything